### DISCUSSION TITLE
Original file vs mp3 transcode

### DISCUSSION THREAD

**Discussion by jhurlbut** [okio-ai/nendo-platform#5] [Q&A]
This is more of a question than an issue but it doesn't look like you have the question github feature activated. I see that an mp3 and the original file is copied to the library directory when you add a track. Is the original file used? If not is there an easy way to stop the original file from being copied into the library to save space?

**Comment by faradox** [comment]
Excellent point. Nendo Core already has that feature, it's just not hooked into the Platform yet. Will add a PR for that and let you know once it's in.

**Reply by jhurlbut** [reply]
Cool. It might be nice to also have some options in the front end for conversion settings: mp3 vs aiff, bitrate etc

**Comment by faradox** [comment]
Ok so the problem with that feature request is that the entire application is running inside docker and has no global access to the host filesystem where the original files reside. So the only way to make this work is to run Nendo Platform directly on your host instead of inside docker for which we currently do not have scripts/recipes. It's pretty straightforward but not as straightforward as calling `make run`. I'll add a note to myself to extend the documentation with a tutorial on how to run a bare metal version of the Nendo Platform but there's many things that can go wrong, depending on your host configuration so we'll declare this as an "experimental" mode for people who know what they are doing.

**Reply by jhurlbut** [reply]
Some basic instructions on running from the source repos instead of docker images would be great
